TsRandomizer/Randomisation/ItemPlacers/ItemLocationRandomizer.cs

```diff
@@ -76,6 +83,7 @@ protected ItemLocationRandomizer(Seed seed, ItemInfoProvider itemInfoProvider, I
 				ItemInfoProvider.Get(EInventoryUseItemType.Antidote),
 				ItemInfoProvider.Get(EInventoryUseItemType.SandBottle),
 				ItemInfoProvider.Get(EInventoryUseItemType.HiSandBottle),
+				ItemInfoProvider.Get(EItemType.MaxSand)
```

Comment:
I dont think this is such a good idea as then you have no idea how many sand ups you get, either way this has nothing todo with the scope of this PR so i suggest to leave this out and maybe discuss/add it as a seperated PR

Reply:
Removed. Slipped through from a test.